docs/source/installation/developer-anaconda.rst

```diff
@@ -11,6 +11,8 @@ Developer installations are useful for those who want to customise the *labscrip
 In this example, we will use an existing conda environment named `py38`.
 Skip the first line/step if continuing on from the instructions to :ref:`set up this environment <installation/setting-up-an-environment:Anaconda Python>`.
 
+.. attention:: 
+    For the following to work correctly on Windows, you need to use the Anaconda Powershell Prompt, not the Anaconda Prompt.
 
```

Comment:
Maybe we can add a link to the relevant github issues?
```suggestion
This will be the case until the following bugs are fixed: `conda-build#3813 <https://github.com/conda/conda-build/issues/3813>`_ and `conda#9445 <https://github.com/conda/conda/issues/9445>`_.

```